packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -74,27 +74,42 @@ export class SlackBot {
         }
 
         try {
-            const slackReceiver = new ExpressReceiver({
-                ...slackOptions,
-                installationStore: {
-                    storeInstallation: (i) =>
-                        this.slackAuthenticationModel.createInstallation(i),
-                    fetchInstallation: (i) =>
-                        this.slackAuthenticationModel.getInstallation(i),
-                    deleteInstallation: (i) =>
-                        this.slackAuthenticationModel.deleteInstallation(i),
-                },
-                logLevel: LogLevel.INFO,
-                app: expressApp,
-            });
-
-            const app = new App({
-                ...slackOptions,
-
-                receiver: slackReceiver,
-            });
+            if (this.lightdashConfig.slack?.appToken) {
+                const app = new App({
+                    ...slackOptions,
+                    installationStore: {
+                        storeInstallation: (i) =>
+                            this.slackAuthenticationModel.createInstallation(i),
+                        fetchInstallation: (i) =>
+                            this.slackAuthenticationModel.getInstallation(i),
```

Comment:
Can we handle the delete here?
```
                        deleteInstallation: (i) =>
                            this.slackAuthenticationModel.deleteInstallation(i),
```

Reply:
yes, its missing and should be in place, good catch 👍